final_task/rss_reader/args_parser.py

```diff
@@ -0,0 +1,73 @@
+"""
+This module process the input arguments
+"""
+
+import argparse
+import re
+import logs
+import sys
+import datetime
+
+
+def get_parse(args_in='') -> dict:
+    """This function takes the arguments from command line"""
+
+    parser = argparse.ArgumentParser(prog='RSSTaker', description='RSS reader. Takes the arguments from command line.')
+
+    parser.add_argument('url', type=str, help='Link to RSS channel(line without spaces). Mandatory for all actions.')
+    parser.add_argument('-v', '--version', action='version', version='%(prog)s 3.1', help='Print version info.')
+    parser.add_argument('-j', '--json', action='store_const', const=True, help='Print result as json in stdout.')
+    parser.add_argument('-b', '--verbose', action='store_const', const=True, help='Print all logs in stdout.')
+    parser.add_argument('-l', '--limit', type=int, default=1, help='Limit of news topics (natural number).')
+    parser.add_argument('-d', '--date', type=int, help='Date to print news from history, yyyymmdd (natural number).')
+
+    if args_in:
+        try:
+            args = parser.parse_args(args_in)
+            logs.log_init_args(args)
+        except SystemExit:
+            logs.log_err_init_args(args_in)
+            logs.log_err_exit()
+            parser.exit()
```

Comment:
Here and below: It is better to have only one exit point in the program. For example you can raise your custom exception here and handle it in the main function.

Reply:
It was fixed for major cycles. The rest will be fixed after fourth iteration.